qa-tests-backend/src/main/groovy/services/ImageService.groovy

```diff
@@ -37,11 +39,15 @@ class ImageService extends BaseService {
 
     static scanImage(String image, Boolean includeSnoozed = true, Boolean force = false) {
         try {
-            return getImageClient().scanImage(ImageServiceOuterClass.ScanImageRequest.newBuilder()
-                    .setImageName(image)
-                    .setIncludeSnoozed(includeSnoozed)
-                    .setForce(force)
-                    .build())
+def req = ImageServiceOuterClass.ScanImageRequest.newBuilder()
+                .setImageName(image)
+                .setIncludeSnoozed(includeSnoozed)
+                .setForce(force)
+                .build();
+                    
+return withRetry(1, 15) {
+    return getImageClient().scanImage(req)
+};
```

Comment:
@dcaravel Let's see if the test passes now. Thanks. I will fix the indentation later on.